src/components/NavBar.js

```diff
@@ -0,0 +1,32 @@
+import { IoIosArrowBack } from 'react-icons/io';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { BsMic } from 'react-icons/bs';
+import { AiOutlineSetting } from 'react-icons/ai';
+
+const Nav = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  return (
+    <nav className="navContainer">
```

Comment:
- [ ] Kindly make your nav take the same width as the rest of the content to match the design guidelines.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/176434679-2ef252bc-2355-427a-8025-fb5218cb3e6d.png) | ![image](https://user-images.githubusercontent.com/24703171/176434937-42874099-4648-4870-84da-626174c515cf.png) |

Reply:
- Yes, you can keep the globe symbol.
- No, the search is in the requirement don't delete it and you can use it as a code you are most proud of.
- Yes, you are free to choose your main color

Reply:
Thank you, I will proceed in this way.

Pedro